prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html

```diff
@@ -11,11 +11,11 @@
         </ng-container>
         <ng-container #alertContent
                       class="alert-content">
-          <p>You have cancelled your login to PRIME</p>
+          <p>You have cancelled your log in to PRIME</p>
           <p>For assistance with PRIME, contact <app-prime-phone></app-prime-phone> or
             <app-prime-support-email></app-prime-support-email>
           <p>
-            For assistance logging in with the BC Services Card app, contact the Helpdesk
+            For assistance logging in with the BC Services Card app, contact the Help desk
```

Comment:
Should that be "Help Desk"?  The edit in `prime-angular-frontend/src/app/modules/gis-enrolment/pages/ldap-information-page/ldap-information-page.component.html` indicated that